interfaces/mount/spec.go

```diff
@@ -43,6 +46,18 @@ func (spec *Specification) AddMountEntry(e Entry) error {
 func (spec *Specification) MountEntries() []Entry {
```

Comment:
I'm grepping through the codebase, and MountEntries seems to be called only from tests. Shouldn't the declashing code be a part of AddMountEntry?

Reply:
I'll fix the backend to use the right thing. I may even move the declash logic to the backend directly.

Reply:
Fixed now, I will look at moving it to the backend